src/installer/pkg/sfx/installers/dotnet-runtime-deps/dotnet-runtime-deps-debian.proj

```diff
@@ -6,7 +6,7 @@
   <ItemGroup>
     <LinuxPackageDependency Include="libc6;libgcc1;libgssapi-krb5-2;libstdc++6;zlib1g"/>
     <LinuxPackageDependency Include="libssl1.0.0 | libssl1.0.2 | libssl1.1 | libssl3" />
-    <KnownLibIcuVersion Include="74;72;71;70;69;68;67;66;65;63;60;57;55;52" />
+    <KnownLibIcuVersion Include="78;77;76;74;72;71;70;69;68;67;66;65;63;60;57;55;52" />
```

Comment:
>78 [](http://example.com/codeflow?start=33&length=2)

If we already adding non -released versions, why can't add more version like 79, 80, 81, 81 ax example?

Reply:
I gave a similar suggestion above, @NikolaMilosavljevic will track it separately. I think there we could also revisit if we need the version guard any longer, and consider switching to feature detection.

Reply:
Future improvement is now tracked in https://github.com/dotnet/runtime/issues/115407